mod_reforged/hooks/states/tactical_state.nut

```diff
@@ -1,4 +1,29 @@
 ::Reforged.HooksMod.hook("scripts/states/tactical_state", function(q) {
+	q.initMap = @(__original) function()
+	{
+		// Feat: the same location will always produce the exact same layout
+		// This has no influence over troop spawning as those will be seeded  by "combatSeed" later on
+		if (this.m.StrategicProperties.MapSeed == 0)	// Other fixed seeds take priority
+		{
+			foreach (party in this.m.StrategicProperties.Parties)
+			{
+				if (party.isLocation())
+				{
+					::Reforged.Math.seedRandom(
+						"RF_FixedLocationSeed",		// Fixed salt, specific to use-case
+						party.getID(),				// Location specific salt
```

Comment:
Are we sure that `party.getID()` is the same on save/load? I know that tactical entities `getID()` is changed on save/load.

Reply:
Right, it was meant to be getTypeID instead. its fixed now